scripts/superdesk-templates/templates.js

```diff
@@ -151,8 +151,6 @@
                 criteria.template_type = type;
             }
 
-            criteria.schedule_desk = null;
```

Comment:
@mdhaman can you confirm that this is not breaking something? This was preventing templates with `schedule_desk` to be listed in list.